ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/filter.vrfs.yml

```diff
@@ -42,5 +45,8 @@ tenants:
         description: This VRF would have been configured because of always_include_vrfs_in_tenants, but it is prevented by filter.deny_vrfs
       - name: VRF5
         description: This VRF will be configured because of always_include_vrfs_in_tenants and it is permitted by filter.allow_vrfs
+        bgp:
```

Comment:
does this mean that without this users will receive a message stating they should add a `vrf_id` ? thats going to be confusing no?

Reply:
It tells the user they need to set the vrf_id and why it is needed. So not really.

Reply:
sure but if the user wanted to not configure EVPN the error message will not suggest that they can disable it using `bgp: false`, instead the user would probably end up removing the address family. My point is that from the error message I don't know if this would be obvious that an option is to use `bgp: false`